query-translator/tests/translate.rs

```diff
@@ -87,11 +89,43 @@ fn test_limit() {
     });
 
     let input = r#"[:find ?x :where [?x :foo/bar "yyy"]]"#;
-    let parsed = parse_find_string(input).expect("parse failed");
-    let mut algebrized = algebrize(&schema, parsed);
-    algebrized.limit = Some(5);
-    let select = query_to_select(algebrized);
-    let SQLQuery { sql, args } = select.query.to_sql_query().unwrap();
+    let SQLQuery { sql, args } = translate(&schema, input, 5);
     assert_eq!(sql, "SELECT `datoms00`.e AS `?x` FROM `datoms` AS `datoms00` WHERE `datoms00`.a = 99 AND `datoms00`.v = $v0 LIMIT 5");
     assert_eq!(args, vec![("$v0".to_string(), "yyy".to_string())]);
 }
+
+#[test]
+fn test_unknown_attribute_keyword_value() {
+    let schema = Schema::default();
+
+    let input = r#"[:find ?x :where [?x _ :ab/yyy]]"#;
```

Comment:
Consider a test where you have a concrete ref attribute (say `:db/ident`) and the query includes a concrete unknown ident.